src/lib/Joi/joiValidation.js

```diff
@@ -0,0 +1,71 @@
+const { createBoard } = require('../utils/common')
+const cache = require('memory-cache')
+const {
+  schemaCreateBoard,
+  schemaUpdateBoard,
+  schemaCreateCard,
+  schemaUpdateCard
+} = require('./joiSchemas')
+
+/**
+ * Checks if provided data is covering with created joi schema
+ * @param {Record<string, unknown>} data
+ * @param {Joi.ObjectSchema<any>} schema
+ * @returns {Record<string, unknown>}
+ */
+const joiValidate = (data, schema) => {
+  // Validate
+  const checkSchema = schema.validate(data)
+
+  // If error throw
+  if (checkSchema.error) throw checkSchema.error
+
+  // Return data
+  return checkSchema.value
+}
+
+/**
+ * Returns Joi schema object
+ * @param {boolean} isCreate - by default the create schema is taken
+ * @returns {Promise<Joi.ObjectSchema<any>>}
+ */
+const checkIfCard = async (isCreate = true) => {
```

Comment:
Please, let give it more descriptive name. Also, I suggest to not writing documentation for simple/self explaining logic.